README.md

```diff
@@ -18,97 +18,39 @@
-- Disabled audio exclusive
-- Disabled unneeded devices
-- Disabled power saving (for personal computers)
-- Disabled performance-hungry security mitigations
-- Automatically enabled MSI mode on all devices
-- Optimized boot configuration
-- Optimized process scheduling
-
-### 🔒 Legal
-Many custom Windows OSes distribute their systems by providing a tweaked ISO of Windows. Not only does it violate [Microsoft's Terms of Service](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_English.htm), but it is also not a safe way to install.
-
-Atlas partnered with the Ameliorated team to provide users with a safer and legal way to install, using [AME Wizard](https://ameliorated.io). With it, Atlas fully complies with [Microsoft's Terms of Service](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_English.htm).
-
+- [Branding](https://docs.atlasos.net/branding/)
+- [FAQ & Common Issues](https://docs.atlasos.net/faq-and-troubleshooting/removed-features/)
+
+Want to learn more about Atlas? Visit our [🌐 Website](https://atlasos.net/) and [📚 Documentation](https://docs.atlasos.net/).
+
+## 🤔 What is Atlas?
+Atlas is an open-source project that enhances Windows by eliminating factors that negatively impact gaming performance. It's the perfect choice for reducing system and network latency, and input lag, while maintaining system privacy and prioritizing performance.
+  
+## 👀 Why Atlas?
+### 🔒 Enhanced Privacy
+Atlas removes all types of tracking embedded within Windows and implements numerous group policies to minimize data collection. However, it cannot ensure the security for things outside the scope of Windows, such as browsers and other third-party applications.
+  
+### 🛡️ Increased Security
+Unlike other custom Windows ISOs, Atlas is safer as it doesn't require downloading a modified Windows ISO from the internet. Atlas also includes optional security features (e.g. Defender, Update) that other custom Windows OSs often strip.
+  
+### ✅ Optimized Performance
+Atlas strikes a balance between performance and compatibility. It implements numerous changes to improve Windows performance without breaking essential features.
+  
+### 🔒 Legal Compliance
+Atlas partners with the Ameliorated team to provide users with a safer and legal way to install, using [AME Wizard](https://ameliorated.io). With it, Atlas fully complies with [Microsoft's Terms of Service](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_English.htm).
+
+### 🌐 Open Source
+All scripts used by Atlas are open-sourced in this repository. Executables included in the playbook are open-sourced in our [utilities](https://github.com/Atlas-OS/utilities) repository. 
+
+The packaged Atlas playbook (`.apbx` - AME Wizard script package) can be viewed as a .7z archive, with the password `malte` (standard for AME Wizard playbooks). This is solely to bypass false flags from antiviruses.
```

Comment:
The packaged Atlas playbook, being an AME Wizard script package (`.apbx`), can be opened as a **.zip** archive using the password `malte` without any obfuscation of files, which is the standard for AME Wizard playbooks. This is implemented solely to prevent false positive detections by antivirus software.

Quick rewrite.